packages/main/src/themes/RatingIndicator.css

```diff
@@ -14,39 +15,74 @@
 	cursor: initial;
 }
 
-:host([readonly]) .ui5-rating-indicator-icon.ui5-rating-indicator-readonly-unselected-icon {
+:host([readonly]) .ui5-rating-indicator-readonly-item-unsel {
 	color: var(--sapContent_UnratedColor);
 }
 
+:host(:not([readonly]):not([disabled])) .ui5-rating-indicator-root:hover {
+	opacity: .9;
+}
+
 :host([_focused]) {
 	outline: var(--sapContent_FocusWidth) var(--sapContent_FocusStyle) var(--sapContent_FocusColor);
 	border-radius: var(--_ui5_rating_indicator_border_radius);
 }
 
 .ui5-rating-indicator-root {
 	outline: none;
+	position: relative;
+}
+
+.ui5-rating-indicator-list {
+	list-style-type: none;
+	display: flex;
+	margin: 0;
+	padding: 0;
 }
 
-.ui5-rating-indicator-icon {
+.ui5-rating-indicator-item {
 	position: relative;
-	color: var(--sapContent_UnratedColor);
-	user-select: none;
+	width: var(--_ui5_rating_indicator_item_width);
+	height: var(--_ui5_rating_indicator_item_height);
+	min-width: var(--_ui5_rating_indicator_item_width);
+    min-height: var(--_ui5_rating_indicator_item_height);
 }
 
-.ui5-rating-indicator-icon.ui5-rating-indicator-active-icon {
-	color: var(--sapContent_RatedColor);
+.ui5-rating-indicator-item:not(:last-child) {
+	margin-inline-end: 0.1875rem;
 }
 
-.ui5-rating-indicator-icon.ui5-rating-indicator-half-icon:before {
-	content: "\2605";
-	position: absolute;
-	top: 0;
-	width: 50%;
+.ui5-rating-indicator-item [ui5-icon] {
+	width: 100%;
 	height: 100%;
+	color: inherit;
+	user-select: none;
+}
+
+.ui5-rating-indicator-item.ui5-rating-indicator-item-sel {
 	color: var(--sapContent_RatedColor);
-	overflow: hidden;
 }
 
-.ui5-rating-indicator-stars-wrapper {
-	display: flex;
+.ui5-rating-indicator-item.ui5-rating-indicator-item-unsel {
+	color: var(--sapContent_UnratedColor);
 }
+
+.ui5-rating-indicator-item.ui5-rating-indicator-item-half {
+	color: var(--sapContent_UnratedColor);
+}
+
+.ui5-rating-indicator-item [ui5-icon].ui5-rating-indicator-half-icon {
+	position: absolute;
+    left: 50%;
```

Comment:
Probably you need to check/test (in case you did not already) if these should be also handled in RTL mode:
- left: 50% here
- left: -50% below